<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client_dialog</class>
 <widget class="QDialog" name="client_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>491</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>440</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>520</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Courier</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ressources.qrc">
    <normaloff>:/icones/logo.png</normaloff>:/icones/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color: white;
}

QLineEdit{
	border: 1px solid grey;
	border-radius: 6px;
	padding-left: 15px;
	height: 35px;
}

QComboBox{
	border: 2px solid white;
	border-radius: 8px;
	padding: 1px 18px 1px 3px;
	background-color: black;
	color: white;
	height: 35px;
	padding-left: 15px;
	font-weight: bold;
	selection-background-color: #2980B9;
}</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>548</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ajouter nouveau client</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>81</y>
     <width>501</width>
     <height>342</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Courier</family>
          <pointsize>15</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_edit_nom">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Courier</family>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Courier</family>
          <pointsize>15</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Entreprise</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_edit_entreprise">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Courier</family>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Courier</family>
          <pointsize>15</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Adresse</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_edit_adresse">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Courier</family>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>-1</number>
      </property>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>Courier</family>
          <pointsize>15</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Contact</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_edit_contact">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Courier</family>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <family>Courier</family>
          <pointsize>15</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_edit_email">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Courier</family>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>440</y>
     <width>261</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="annule_buton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Courier</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #585858;
	color: white;
	border: none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 15px
}</string>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ok_bouton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Courier</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #34D481;
	color: white;
	border: none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 15px
}</string>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
